functions/prompt_for_confirmation/lambda_function.py

```diff
@@ -52,7 +52,7 @@ def handle_state(context, receiver, target_type, target, text, prompt_text='', y
     execution_id = context.get('execution_id')
     if resp.data['ok']:
         socless_dispatch_outbound_message(receiver, message_id, investigation_id, execution_id, resp.data)
-        return {"response": resp.data, "message_id": message_id}
+        return {"response": resp.data, "message_id": message_id, "slack_id" : target_id}
```

Comment:
is `slack_id` always going to be a user_id? if so let's name it accordingly. Or is there a benefit to the generic slack_id term?

Reply:
"slack_id" is used when no formatting needs to be done and it can be passed straight through to the API. So it includes formatted channel_names as well

Reply:
👍